### PR TITLE
Revert _FINISHED protocol messages

### PR DESCRIPTION
**Pull request by snario**

### CODE REVIEW

File: packages/cf-adjudicator-contracts/package.json [hunk 1]
```diff
@@ -32,7 +32,7 @@
     "solidity"
   ],
   "devDependencies": {
-    "@counterfactual/types": "0.0.40",
+    "@counterfactual/types": "0.0.39",
```
Comment (cf19drofxots) [inline packages/cf-adjudicator-contracts/package.json]: Change this to `0.0.41` - if it's `0.0.39` then the next change that bumps this to `0.0.40` will conflict with the current published npm package.

File: packages/node/src/methods/app-instance/propose-install/controller.ts [hunk 2]
```diff
@@ -53,6 +47,12 @@ export default class ProposeInstallController extends NodeController {
     params: Node.ProposeInstallParams
   ) {
     const { store, publicIdentifier, networkContext } = requestHandler;
+    const { initialState } = params;
```
Comment (cf19drofxots) [inline packages/node/src/methods/app-instance/propose-install/controller.ts]: This change should not be reverted as we wouldn't want to queue an invalid request.

Comment (snario) [reply packages/node/src/methods/app-instance/propose-install/controller.ts]: It should be in `beforeExecution`

Comment (cf19drofxots) [reply packages/node/src/methods/app-instance/propose-install/controller.ts]: That doesn't make sense though - why would you queue a request that's invalid to begin with?

Comment (snario) [reply packages/node/src/methods/app-instance/propose-install/controller.ts]: `beforeExecution` runs before the queue is added

**Pull request merged**